examples/run_largecyc_CPMG.py

```diff
@@ -188,10 +179,11 @@
     ["ph1", "ph2", "ph_overall", "nEcho", "t2"],
     [len(ph1), len(ph2), len(ph_overall), config_dict["nEchoes"], -1],
 )
-data.setaxis("nEcho", r_[0 : config_dict["nEchoes"]]).setaxis(
-    "ph1", ph1 / 4
-).setaxis("ph2", ph2 / 4).setaxis("ph_overall", ph_overall / 4)
-data.set_prop("postproc_type", "spincore_generalproc_v1")
+data.set_units("t2", "s")
+data.setaxis("nEcho", r_[0 : config_dict["nEchoes"]]).setaxis("ph1", ph1 / 4).setaxis(
+    "ph2", ph2 / 4
+).setaxis("ph_overall", ph_overall / 4)
+data.set_prop("postproc_type", "spincore_largecyc_CPMG_v1")
```

Comment:
increment post processing version so that we aren't multiplying by nScans anymore

Reply:
also we don't have a postprocessing function yet for this guy